src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -1168,7 +1179,7 @@ async Task ITokenCacheInternal.RemoveAccountAsync(IAccount account, Authenticati
                             account: account,
```

Comment:
It is difficult to acquire the account associated with the tokens since there may be multiple accounts associated with the same cache key. Lets discuss tomorow during our sync.

@bgavrilMS

Reply:
Discussed offline: Passing null into notification args